app/models/miq_request.rb

```diff
@@ -615,6 +615,10 @@ def canceled?
     cancelation_status == CANCEL_STATUS_FINISHED
   end
 
+  def initiator
```

Comment:
Would `initiator_type` be a better name?  When I see `MiqRequest#initiator` I would expect to get an AR object back, not a string.